vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -139,34 +139,23 @@ void CompositionBaseComponentView::handleCommand(std::string const &commandName,
 }
 
 int64_t CompositionBaseComponentView::sendMessage(uint32_t msg, uint64_t wParam, int64_t lParam) noexcept {
-  if (msg == WM_KEYUP || msg == WM_KEYDOWN) {
-    facebook::react::KeyboardEvent event;
-    BYTE bKeys[256];
-    if (GetKeyboardState(bKeys)) {
-      if (bKeys[VK_LSHIFT] & 0x80 || bKeys[VK_RSHIFT] & 0x80) {
-        event.shiftKey = true;
-      }
-      if (bKeys[VK_LCONTROL] & 0x80 || bKeys[VK_RCONTROL] & 0x80) {
-        event.ctrlKey = true;
-      }
-      if (bKeys[VK_LMENU] & 0x80 || bKeys[VK_RMENU] & 0x80) {
-        event.altKey = true;
-      }
-
-      event.metaKey = (GetKeyState(VK_LWIN) < 0) || (GetKeyState(VK_RWIN) < 0);
-    }
+  return 0;
```

Comment:
Should we maybe log an unsent message here?

Reply:
Right now there are a lot of messages going through here... once more of the input and other message handing has been updated to not get routed using messages, then this can probably be removed entirely. -- It would be a very noisy log right now.  -- Will consider after at least pointer messages no longer show up here.